doc/en/installation/from_packages.rst

```diff
@@ -21,153 +21,144 @@ SELinux should be disabled; for this, you have to modify the file */etc/selinux/
 .. note::
     After saving the file, please reboot your operating system to apply the changes.
 
+A quick check of SELinux status:
+
+::
+
+    $ getenforce
+    Disabled
+
+
 ******************
-Repository install
+Repository installation
 ******************
 
```

Comment:
For title tags you need to put '*' until the last charectère

Reply:
done